codecs/build.sbt

```diff
@@ -1,4 +1,14 @@
+scalaCheckVersion := {
+  scalazVersion.value match {
+    case VersionNumber(Seq(7, 2, _*), _, _) => "1.12.6"
+    case VersionNumber(Seq(7, 1, _*), _, _) =>
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((2, n)) if n <= 11 => "1.11.4"
+        case Some((2, n)) => "1.11.6"
+      }
+  }
+}
```

Comment:
This is copy-pasted because sbt-rig sets this key in `versionSettings` instead of as a global default.  We could DRY this up with a QuiverPlugin on top of rig.

I didn't just upgrade to 1.13, because there isn't a scalacheck binding for 1.13 for scalaz 7.1.  We walk this tightrope to use the versions that our scalazes were built against.